### PR TITLE
Renamed some inconsistently named sliders and the 'Other metals' slide.

### PR DESCRIPTION
**Pull request by AlexanderWirtz**
Minor changes. addresses issue #1891

### CODE REVIEW

**Comment by AlexanderWirtz** [comment]
assigning to @jorisberkhout for good measure

File: config/locales/nl_input_elements.yml [hunk 1]
```diff
@@ -144,9 +144,9 @@ nl:
     households_number_of_inhabitants_chart: "Aantal inwoners"
     households_inhabitants_per_household: "Aantal inwoners per huishouden"
     households_replacement_of_existing_houses: "Vervanging bestaande huizen"
-    households_space_heater_heatpump_air_water_electricity_share: "Warmtepomp (lucht)"
-    households_water_heater_heatpump_air_water_electricity_share: "Warmtepomp (lucht)"
-    households_water_heater_heatpump_ground_water_electricity_share: "Warmtepomp (grond)"
+    households_space_heater_heatpump_air_water_electricity_share: "Luchtwarmtepomp"
+    households_water_heater_heatpump_air_water_electricity_share: "Luchtwarmtepomp"
+    households_water_heater_heatpump_ground_water_electricity_share: "Warmtepomp (bodem)"
```
Comment (jorisberkhout) [inline config/locales/nl_input_elements.yml]: @AlexanderWirtz should we not change this to 'Bodemwarmtepomp' for consistency with the 'Luchtwarmtepomp'?

**Comment by AlexanderWirtz** [comment]
I wondered the same, but shied away from doing so.

**Comment by jorisberkhout** [comment]
Ok! Merging then.

**Comment by AlexanderWirtz** [comment]
Don't care that much. Perhaps we can have a vote on this? We need a third vote

**Comment by jorisberkhout** [comment]
Ok, let's ask @ChaelKruip what he thinks! Or anyone that reads this issue before @ChaelKruip does ;)

**Comment by ChaelKruip** [comment]
I like `Bodemwarmtepomp`.

**Pull request merged**